http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3004,7 +3012,7 @@ components:
             workerDiskSize:
               type: integer
               description: >
-                Optional, the size in gigabytes of the disk on the master node.
+                Optional, the size in gigabytes of the disk on the worker nodes.
```

Comment:
Follow-up question here:

How does this parameter interact with `numberOfWorkerLocalSSDs` ?

Reply:
See https://cloud.google.com/dataproc/docs/reference/rest/v1/ClusterConfig#DiskConfig

`workerDiskSize` is passed to the Dataproc `bootDiskSizeGb` field and `numberOfWorkerLocalSSDs` is passed to `numLocalSsds`. 

I'm not completely confident in our support for `numberOfWorkerLocalSSDs` btw -- I don't think Terra UI exposes that option.